spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration.java

```diff
@@ -106,11 +110,13 @@ public FlywaySchemaManagementProvider flywayDefaultDdlModeProvider(
 		private List<FlywayCallback> flywayCallbacks;
 
 		public FlywayConfiguration(FlywayProperties properties,
```

Comment:
Shouldn't we be retaining the original constructor for binary compatibility?

Reply:
This will be a 2.0 enhancement so this is fine.